README.md

```diff
@@ -1,10 +1,76 @@
-# Picover
+# Picover [![Dev branch verification](https://github.com/intive/Picover/actions/workflows/default_branch_verification.yml/badge.svg)](https://github.com/intive/Picover/actions/workflows/default_branch_verification.yml) [![Pull Request verification](https://github.com/intive/Picover/actions/workflows/pull_request_verification.yml/badge.svg)](https://github.com/intive/Picover/actions/workflows/pull_request_verification.yml)
```

Comment:
i see two commits in this pr :D

Reply:
Yes, I rebased the last branch which I made the current branch and I forgot to rebase :P